app/src/lang/translations/ar-SA.yaml

```diff
@@ -267,8 +267,8 @@ create_translations: إنشاء ترجمات
 auto_refresh: تحديث تلقائي
 refresh_interval: الفاصل الزمني للتحيين
 no_refresh: المتابعة على أية حال
-refresh_interval_seconds: تحديث فوراً <unk> كل ثانية <unk> كل {seconds} ثانية
-refresh_interval_minutes: كل دقيقة <unk> كل {minutes} دقيقة
+refresh_interval_seconds: كل | كل ثانية | تحديث فورًا {seconds} ثانية
```

Comment:
I've already fixed that upstream on Crowdin 👍 So all changes to this translation are to be removed from this PR.

Reply:
Ah lovely 👍🏻

Reply:
They are multiple values (based on the passed-in count), just like other [langs](https://github.com/directus/directus/blob/1b0ea954d35a13377c994ab1d2bc4808e5fdaaec/app/src/lang/translations/en-US.yaml#L324)

Reply:
Copy that! We'll have to change those in locales.directus.io, otherwise it's gonna be synced-in wrong again 🙂  (@paescuj mind taking a look there to see if it's correct now?)

Reply:
Yep, correct on Crowdin ✅ Will land in main with [#20507 (app/src/lang/translations/ar-SA.yaml)](https://github.com/directus/directus/pull/20507/files#diff-25db3b67b66dd06b2d9e21c77f331f4cf67b90b381df6c9d21b72f19c9021763).